src/rules/no_default_default.rs

```diff
@@ -0,0 +1,79 @@
+use crate::{rule, rule::Rule, rule_listener, ViolationBuilder};
+
+pub fn no_default_default_rule() -> Rule {
+    rule! {
+        name => "no_default_default",
+        fixable => true,
+        create => |_context| {
+            vec![
+                rule_listener! {
+                    query => r#"(
+                      (call_expression
+                        function:
+                          (scoped_identifier
+                            path:
+                              (identifier) @first (#eq? @first "Default")
+                            name:
+                              (identifier) @second (#eq? @second "default")
+                          )
+                      ) @c
+                    )"#,
+                    capture_name => "c",
+                    on_query_match => |node, query_match_context| {
+                        query_match_context.report(
+                            ViolationBuilder::default()
+                                .message(r#"Use '_d()' instead of 'Default::default()'"#)
+                                .node(node)
+                                .fix(|fixer| {
+                                    fixer.replace_text(node, "_d()");
```

Comment:
What's that this breaks your code?

I think the vision was that a fixer could also add a corresponding `use` declaration if not already present (in a known-imports-y way)

But haven't done that yet

(to be specific I don't mean that adding `use`'s gets "baked into" the `Fixer`, I mean that it is within the power of what you as a rule author can do with a `Fixer` (+ a `tree_sitter::Node`) to navigate the entire file's tree-sitter AST and detect whether it's already `use`'d and if not apply an additional fix to add it in an appropriate-looking place)

Reply:
Ah, that starts sounding a bit tricky at least in Rust, if the user isn't using the unstable rustfmt option to rewrite your imports nicely. (It'd be annoying if it added it in the "wrong" style.)